pkg/bwagreement/server_test.go

```diff
@@ -113,9 +96,19 @@ func NewTestServer(t *testing.T) *TestServer {
 	return ts
 }
 
+var (
+	// for travis build support
+	testPostgres = flag.String("postgres-test-db", os.Getenv("STORJ_POSTGRESKV_TEST"), "PostgreSQL test database connection string")
+)
+
 func newTestServerStruct(t *testing.T) *Server {
-	psqlInfo := getPSQLInfo()
-	s, err := NewServer("postgres", psqlInfo, zap.NewNop())
+
+	if *testPostgres == "" {
+		//hard coded for local default, run locally --> psql -c 'create database pointerdb' -U postgres
+		*testPostgres = "postgres://postgres@localhost/pointerdb?sslmode=disable"
```

Comment:
Do not hard code this, skip the test with an error. See the example how to do this. You may end up destroying someones local database this way.

Reply:
removed and made change as suggested